### DISCUSSION TITLE
Generic "File" icon

### DISCUSSION THREAD

**Discussion by HartiganHM** [twilio-labs/paste#1581] [Request: Icon] [answered]
Hey, team!

We need a generic "File" icon for a multiple-file attachment input we're working on. Happy to take any suggestions, but below is what I found on Streamline. Our initial UI mocks are below as well. We have ~8-10 file types we're currently accepting, so we figured a generic icon would be best, but let us know if you have any thoughts!

> Icon from Streamline
> ![image](https://user-images.githubusercontent.com/24458700/122478842-5778c800-cf87-11eb-9efc-da856ac53f2a.png)

> UI Mocks
> ![image](https://user-images.githubusercontent.com/24458700/122478862-63fd2080-cf87-11eb-9a42-a75e7114e76a.png)

**EDIT**

I did just notice the `CustomIcon`. Would this work?

![image](https://user-images.githubusercontent.com/24458700/122478972-960e8280-cf87-11eb-8b8f-c6cc5b05b572.png)

**Comment by serifluous** [comment, marked as answer]
@HartiganHM Flex is using this icon right now to show a "file":
<img width="262" alt="Screen Shot 2021-06-17 at 3 44 16 PM" src="https://user-images.githubusercontent.com/3821840/122481235-301ffc00-cf83-11eb-9fc9-c8c46136b742.png">

It looks pretty close to what you shared. Would something like that work?

**Reply by SiTaggart** [reply]
Bump @HartiganHM , would the icon Sarah shared worked for you?

**Reply by HartiganHM** [reply]
Hey, all! Sorry for the delay I was on PTO. @serifluous that looks great for us! We would be using it for image files as well, so I'm not sure if that changes the approach.

**Comment by kate-porter** [comment]
Hey @HartiganHM! Sorry for the delay on this icon, but we are finally getting to it 🎉

How does this look? Would this work for all your use cases?
![image](https://user-images.githubusercontent.com/5374045/129249016-57391559-694c-42d8-8124-c0bb026505b3.png)

**Comment by gloriliale** [comment]
Hi @HartiganHM! We've added both Attachment and File icons! Feel free to use whichever you think fits your needs best.
